gcp/live/dev/k8s/stackdriver-exclusions/calico-typha-health-summary/terraform.tfvars

```diff
@@ -0,0 +1,24 @@
+# ↓ Module metadata
+terragrunt = {
+  terraform {
+    source = "/project/modules//gcp-stackdriver-exclusion"
+  }
+
+  dependencies {
+    paths = [
+      "../../cluster",
+    ]
+  }
+
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+exclusion_name = "calico-typha-health-summary-ok"
+exclusion_description = "calico typha 'Overall health summary' that looks ok"
+# TODO: Better (multi-line) string formatting
+# TODO: How to get project name into logName? Use ":/logs/kubelet" instead of exact match? Old version: 'logName=projects/gpii-gcp-dev-tyler/logs/kubelet'
+# NOTE: Trailing space on ' 200 ' is important!
+exclusion_filter = "resource.type=container AND logName:/logs/calico-typha AND textPayload:\"Overall health summary=&health.HealthReport{Live:true, Ready:true}\""
```

Comment:
Not sure how the `logName` is constructed, but at least Google's UI seems to prefer
```
resource.labels.container_name="calico-typha"
```
over 
```
logName:/logs/calico-typha
```

While probably mainly visual, I would expect exact value match to be faster than search for a contained string).

Reply:
I think efficiency of these filters is somewhat important. Google has a doc with some suggestions: https://cloud.google.com/logging/docs/view/advanced-filters#finding-quickly

I verified that `logName` is indexed, but you are correct that exact string matches are preferred over substring matches and I think I missed `resource.labels.container_name`.

However, it isn't clear to me whether `resource.labels.container_name` is indexed -- is `container_name` one of the "[indexed resource labels in resource.labels](https://cloud.google.com/logging/docs/view/advanced-filters#indexed-fields)"?

Anyway, I'll revisit this when I take another look at which calico logs we want to exclude. Thanks for the suggestion.